browser/components/CodeEditor.js

```diff
@@ -644,6 +618,57 @@ export default class CodeEditor extends React.Component {
     }
   }
 
+  setCodeEditorLintConfig () {
+    const { mode } = this.props
+    const checkMarkdownNoteIsOpening = mode === 'Boost Flavored Markdown'
```

Comment:
Nit: I think renaming the const would be good. As it's not opening - it is already open. Or is it about to open?
```suggestion
    const checkMarkdownNoteIsOpen = mode === 'Boost Flavored Markdown'
```

Reply:
Thnak you for review!
It's a false by my wrong English.